homeassistant/components/openweathermap/const.py

```diff
@@ -146,7 +147,7 @@
     "rainy": [300, 301, 302, 310, 311, 312, 313, 500, 501, 520, 521],
     "snowy": [600, 601, 602, 611, 612, 620, 621, 622],
     "snowy-rainy": [511, 615, 616],
-    "sunny": [800],
+    "sunny/clear-night": [800],
```

Comment:
Should `WEATHER_CODE_SUNNY_OR_CLEAR_NIGHT` be used on this line?

Reply:
These are the supported conditions: https://developers.home-assistant.io/docs/core/entity/weather#recommended-values-for-state-and-condition

New ones means a change to the entity model and should be suggested via the architecture repository.

Reply:
I didn't mean a new state, but to use the constant instead of the integer.

Reply:
```suggestion
    "sunny/clear-night": [WEATHER_CODE_SUNNY_OR_CLEAR_NIGHT],
```